packages/core/src/components/DataTable/defaultContentRenderers/index.tsx

```diff
@@ -0,0 +1,11 @@
+import TextInputRenderer from './TextInputRenderer';
+import TextRenderer from './TextRenderer';
+import { RendererProps } from '../types';
+
+export default function renderDefaultContent(props: RendererProps) {
+  if (props.editMode) {
+    return TextInputRenderer({ ...props });
```

Comment:
`<TextInputRenderer {...props} />`

Reply:
I messed up months ago and called an argument `key` and now I can't use named props because it's a reserved. I could change `key` everywhere, but it's a lot and this seems reasonable to me

Reply:
In that case, can you leave a comment above on why it's that way.